src/app/space/create/codebases/codebases-item-actions/codebases-item-actions.component.ts

```diff
@@ -76,10 +81,47 @@ export class CodebasesItemActionsComponent implements OnDestroy, OnInit {
   }
 
   /**
+   * Confirmation dialog for codebase removal.
+   *
+   * @param {MouseEvent} event mouse event
+   */
+  confirmDeleteCodebase(event: MouseEvent): void {
+    event.stopPropagation();
+    this.dialog = {
+      'title': 'Confirm codebase deletion',
+      'message': 'Are you sure you want to deleteCodebase codebase?',
```

Comment:
Change to 'You are about to remove a codebase from you Space. Please verify this action by clicking Remove below.'

This statement matches others in OpenShift.io that are used when removing/deleting an item.